### ISSUE TITLE
Build failure: vscode-extensions.vadimcn.vscode-lldb

### ISSUE THREAD

**Issue by becknik** [NixOS/nixpkgs#266353]
### Steps To Reproduce

```console
nix-shell -p vscode-extensions.vadimcn.vscode-lldb
```

### Build log

[personal gist](https://gist.github.com/becknik/cee851ab390cf20eaf293b20c8b1da84)

### Additional context

Original issue (now closed) with same built failure on Darwin aarch64 from a few months ago: #241235
Seems to be MacOS-specific...

### Notify maintainers

@nigelgbanks 
I'd really appreciate your thoughts on this, I'm basically a newbie to Nix :/

### Metadata

```console
$ nix-shell -p nix-info --run "nix-info -m" 
 - system: `"x86_64-darwin"`
 - host os: `Darwin 23.1.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.17.1`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/Users/jbecker/.nix-defexpr/channels/nixpkgs`
```

**Comment by hurricanehrndz** [comment]
You can use this patch https://github.com/NixOS/nixpkgs/pull/211321/files

**Comment by superherointj** [comment]
Fixed by #211321

Re-open if issue persist.
